scripts/lib/LinkChecker.ts

```diff
@@ -80,18 +80,24 @@ export class Link {
   }
 
   /**
-   * True if link is valid, otherwise false
+   * Returns a string with the error found.
    */
-  async checkExternalLink(): Promise<boolean> {
-    const response = await fetch(this.value, {
-      headers: { "User-Agent": "prn-broken-links-finder" },
-    });
-
-    if (response.status >= 300) {
-      return false;
+  async checkExternalLink(): Promise<string> {
+    try {
+      const response = await fetch(this.value, {
+        headers: { "User-Agent": "prn-broken-links-finder" },
+      });
+
+      if (response.status >= 300) {
+        return "Could not find link '" + this.value + "'";
```

Comment:
You can use string interpolation. Ditto down below.

```suggestion
        return `Could not find link '${this.value}'";
```